### PR TITLE
fix test, forget adding the test suite and miniredis has no INFO command

### PR DESCRIPTION
**Pull request by siddontang**

### CODE REVIEW

File: pkg/utils/redis_utils_test.go [hunk 1]
```diff
@@ -42,7 +44,8 @@ func (s *testUtilsSuite) TestPing(c *C) {
 	c.Assert(err, IsNil)
 }
 
-func (s *testUtilsSuite) TestGetInfo(c C) {
-	_, err := GetRedisInfo(s.addr, "", s.auth)
-	c.Assert(err, IsNil)
+func (s *testUtilsSuite) TestGetInfo(c *C) {
+	// miniredis has no INFO command, we will use qdb instead later.
```
Comment (xiang90) [inline pkg/utils/redis_utils_test.go]: not quite sure why you commented out all the test code? to "fix" this test?

Comment (siddontang) [inline pkg/utils/redis_utils_test.go]: miniredis has not INFO command, so we can not test some functions with it.

Later @qiuyesuifeng will replace miniredis with qdb, atm, we can re-add this test.

Comment (xiang90) [inline pkg/utils/redis_utils_test.go]: we usually use http://golang.org/pkg/testing/#T.SkipNow to do this kind of thing. Just for your information.

Comment (siddontang) [inline pkg/utils/redis_utils_test.go]: Great!

**Comment by qiuyesuifeng** [comment]
LGTM.

**Pull request merged**